src/main/java/com/google/api/codegen/transformer/csharp/CSharpGapicClientTransformer.java

```diff
@@ -133,7 +127,7 @@ private StaticLangApiAndSettingsFileView generateApiAndSettingsView(
 
     // must be done as the last step to catch all imports
     csharpCommonTransformer.addCommonImports(context);
-    fileView.fileHeader(fileHeaderTransformer.generateFileHeader(context));
+    fileView.fileHeader(fileHeaderTransformer.generateFileHeader(context, importTypeTransformer));
```

Comment:
I would prefer the fileHeaderTransformer to take importTypeTransformer in its constructor.